pragmaticcoders/docker-geth-dev
# Make it easier to use geth for RPC testing locally

## What did I do:
* Add in makefile for basic commands `build`, `rpc`, `kill`, `test`
  * `build` creates the docker container.
  * `rpc` starts geth, if you look at changes to rungeth.docker, I check if your guy's "fast miner" is present, if not, just start it in RPC mode. I can change this if this doesn't work for your main branch.
  * `kill` shuts down the geth container.
  * `test` does the curl command. I can change this in the future.

* Update documentation. The documentation says the accounts are seeded with ether. The genesis.json seeds in **wei**. I confirmed this by using [web3.getBalance](https://github.com/ethereum/wiki/wiki/JavaScript-API#returns-32). 

## How to QA
* Check if this works with your fast miner.
* Run the make commands.

## TODO
 * Add in `make genesis` command which would allow you to easily change the genesis.json and run the server.
